src/main/java/TaskList.java

```diff
@@ -0,0 +1,150 @@
+                            + " To: " + tasks[i-1].getEventEnd() + ")");
+                    continue;
+
+                default:
+                    System.out.println(" (ERROR: OTHER TYPE)");
+            }
+        }
+        Terri.printDivider();
+    }
+
+    public static void addToDo(String newToDo) {
+        if (checkTasklistCapacity()) {
+            return;
+        }
+        tasks[taskCounter++] = new ToDo(newToDo);
+        System.out.println("Just added: " + newToDo + " to your list as a ToDo!");
+        printNumberOfTasks();
+        Terri.printDivider();
+    }
+
+    // Parse deadline information from user input and log Deadline
+    public static void handleDeadline(String[] keyWord) {
+        String newBy = null;
+        String newDeadline = null;
+        StringBuilder tempDeadlineInfo = new StringBuilder();
+
+        // Iterate through user input to concatenate
+        // deadline description/date information
+        for (int i = 1; i <= keyWord.length; i++) {
+            if (keyWord[i].equals("/by")) {
+                newBy = Terri.extractSubArray(keyWord, i+1, keyWord.length);
+                newDeadline = tempDeadlineInfo.toString().trim();
+                break;
+            }
+            tempDeadlineInfo.append(keyWord[i]).append(" ");
+        }
+
+        TaskList.addDeadline(newDeadline, newBy);
+    }
+
+    public static void addDeadline(String newDeadline, String newBy) {
+        if (checkTasklistCapacity()) {
+            return;
+        }
+        tasks[taskCounter++] = new Deadline(newDeadline, newBy);
+        System.out.println("Just added: '" + newDeadline + "' to your list as a Deadline!");
+        printNumberOfTasks();
+        Terri.printDivider();
+    }
+
+    // Parse event information from user input and log event
+    public static void handleEvent(String[] keyWord) {
+        int startIdx = 0;
+        int endIdx = 0;
+
+        // Locate time information in user-input
+        for (int i = 0; i < keyWord.length; i++) {
+            if (keyWord[i].equals("/from")) startIdx = i;
+            if (keyWord[i].equals("/to")) {
+                endIdx = i;
+                break;
+            }
+        }
+
+        String newDescription = Terri.extractSubArray(keyWord, 1, startIdx);
+        String newStart = Terri.extractSubArray(keyWord, startIdx + 1, endIdx);
+        String newEnd = Terri.extractSubArray(keyWord, endIdx + 1, keyWord.length);
+
+        TaskList.addEvent(newDescription, newStart, newEnd);
+    }
+
+
+    public static void addEvent(String newEvent, String From, String To) {
+        if (checkTasklistCapacity()) {
+            return;
+        }
+        tasks[taskCounter++] = new Event(newEvent, From, To);
+        System.out.println("Just added: '" + newEvent + "' to your list as an Event!");
+        printNumberOfTasks();
+        Terri.printDivider();
+    }
+
+    // Updates task isDone field to be completed
```

Comment:
Good job in commenting your code! Just a suggestion, you may look into using JavaDoc format for function comments instead